pages/common/linkchecker.md

```diff
@@ -0,0 +1,20 @@
+# linkchecker
+
+> Command line client to check HTML documents and websites for broken links.
+> More information: <https://linkchecker.github.io/linkchecker/>.
+
+- Find broken links on https://example.com/:
+
+`linkchecker {{https://example.com/}}`
+
+- Also check external URLs:
```

Comment:
I assume you mean URLs that point to other domains? Perhaps this will read better?
```suggestion
- Also check URLs that point to external domains:
```